download_and_create_reference_datasets/v03/write_combined_interval_reference_ht.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+import argparse
+import os
+
+import hail as hl
+
+from hail_scripts.reference_data.combine import get_ht, join_hts, update_joined_ht_globals
+from hail_scripts.reference_data.config import GCS_PREFIXES
+from hail_scripts.utils.hail_utils import write_ht
+
+DATASETS = ['gnomad_non_coding_constraint', 'screen']
+INTERVAL_REFERENCE_HT_PATH = 'combined_interval_reference/combined_interval_reference.GRCh{genome_version}.ht'
+VERSION = '1.0.0'
+
+def update_existing(destination_path: str, dataset: str, genome_version: str):
+    destination_ht = hl.read_table(destination_path)
+    dataset_ht = get_ht(dataset, genome_version)
+    destination_ht = (destination_ht
+        .drop(dataset)
+        .join(dataset_ht, 'outer')
+        .filter(
```

Comment:
I updated this beyond the logic we have in `join_hts` to also filter out rows where every dataset is null.  

I'm making the assumption that, in addition to potentially adding rows with an updated dataset we might also be removing rows.  If the rows we're removing have presence in other datasets we want to keep them, but if they are unique to the old version of the dataset we want to remove them.  Is this assumption correct?

Reply:
I'm not sure theres much harm in keeping rows with no datasets, especially if we are running VEP on the whole dataset. @mike-w-wilson do we anticipate any meaningful number of rows ever dropping form a dataset? My gut feeling is that this would be a pretty rare event but maybe I'm wrong